README.md

```diff
@@ -97,7 +97,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
 
 # License
 
-Copyright 2021-2023 SUSE LLC
+Copyright 2021-2024 SUSE LLC
 
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use
 this file except in compliance with the License. You may obtain a copy of the
```

Comment:
since we're touching it, this is something I ought to do everywhere else:
```suggestion
this code repository except in compliance with the License. You may obtain a copy of the
```